lib/docbuild.js

```diff
@@ -0,0 +1,10 @@
+var createDocs = require('node-offline-api').createDocs
+var buildOptions = require('node-offline-api').buildOptions
+var fs = require('fs')
+
+buildOptions.buildDir = process.cwd()
+buildOptions.buildName = 'node_apidoc'
+
+fs.stat('../node-apidoc', function (err) {
+  if (err) { createDocs() }
```

Comment:
There can theoretically be various errors here. I think the [`.code === "ENOENT"`](https://nodejs.org/api/errors.html#errors_common_system_errors) is the error we are looking for.